tools/test-setup.sh

```diff
@@ -41,7 +41,7 @@ if [ -f /etc/os-release ]; then
                 18.04)
                     # ubuntu xenial vagrant is too old so it doesn't support triggers, used by the alpine box
                     sudo apt-get remove --purge -y vagrant
-                    wget https://releases.hashicorp.com/vagrant/2.2.9/vagrant_2.2.9_x86_64.deb
+                    wget --no-show-progress https://releases.hashicorp.com/vagrant/2.2.9/vagrant_2.2.9_x86_64.deb
```

Comment:
btw, maybe update to 2.2.10 here too, as done with rpm distros ?

Reply:
Can be done later, this already brew more than needed.